kod/object/active/holder/nomoveon/battler/player/user.kod

```diff
@@ -2944,7 +2969,7 @@ messages:
 
       % Check for cheating by sending more move packets than allowed.
       % Don't log too many cheat detections, because it'll fill up our logs.
-      if piMovesCounter > MOVEMENT_COUNT_THRESHOLD
+      if piMovesCounter > piMoveCountThreshold
```

Comment:
There's a speedhack detection component of this effort that I overlooked in my work here: `iDelta` a few lines above. 

I believe this needs to scale in relation to the new server values to properly apply the decay, like so:
```
% Number of moves “consumed” due to the passage of time
decayFactor = (1000/MOVE_INTERVAL));

piMovesCounter = (piMovesCounter + 1) - (iDelta * decayFactor);
```

To do this, I'll also need to grab move interval from the config and that has me wondering: should we store these in `System` and then retrieve and store them on `User` during logon?